pbs_light.go

```diff
@@ -211,14 +211,37 @@ func (deps *cookieSyncDeps) CookieSync(w http.ResponseWriter, r *http.Request, _
 	defer r.Body.Close()
 
 	csReq := &cookieSyncRequest{}
-	err := json.NewDecoder(r.Body).Decode(&csReq)
+	csReqRaw := map[string]json.RawMessage{}
+	err := json.NewDecoder(r.Body).Decode(&csReqRaw)
 	if err != nil {
 		if glog.V(2) {
 			glog.Infof("Failed to parse /cookie_sync request body: %v", err)
 		}
 		http.Error(w, "JSON parse failed", http.StatusBadRequest)
 		return
 	}
+	if UUID, ok := csReqRaw["uuid"]; ok {
```

Comment:
I'm pretty confused... you're parsing this, but the value isn't used anywhere. What's the point here?

Reply:
I need to parse it of I am going to pass it through. If we a deprecating it then I can not do it. :) Just grabbing it raw leaves in the quotes around the string.

Reply:
Oh, you are correct that I never put it in `csReq`. But now I have removed `UUID` from the struct and it is now a one element list. Should I keep the struct (balances against the response struct, and eases expanding the API later) or should I just pull the struct and just work with a bidders list?